cypress/integration/google.spect.js

```diff
@@ -0,0 +1,6 @@
+describe("This is my first cypress test", () => {
+    it("should have a title", () => {
+        cy.visit("https://www.google.com/");
+        cy.title().should("eq", "Google");
+    });
+});
```

Comment:
Please add one empty line at the end of the file to remove this warning:
![image](https://user-images.githubusercontent.com/10209180/166987371-c133d9ca-ed37-4880-8c00-28f85bc2e0c8.png)